src/Orleans/Facet/FacetAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using System;
+
+namespace Orleans
+{
+    /// <summary>
+    /// Marker attribute for facets
+    /// </summary>
+    public abstract class FacetAttribute : Attribute
```

Comment:
I'll change the marker to an interface.  I considered (and prototyped) both at various times and settled on the abstract attribute base class simply because it did not require me to write a filter for the custom attributes (which is admittedly trivial).